client/packages/common/src/ui/layout/tables/material-react-table/useBaseMaterialTable.tsx

```diff
@@ -77,7 +77,10 @@ export const useBaseMaterialTable = <T extends MRT_RowData>({
 
     data: processedData,
     enablePagination: false,
+
+    layoutMode: 'grid',
```

Comment:
grid here, along with `grow: true` in `useMaterialTableColumns`, means columns will grow to fit full screen width, for tables with fewer columns